en/publish/publish-wechatgame.md

```diff
@@ -1,114 +1,99 @@
+<iframe frameborder="0" width="100%" height="440" src="https://v.qq.com/iframe/player.html?vid=c05255stri7&tiny=0&auto=0" allowfullscreen></iframe >
 {% endraw %}
 
-## 微信小游戏平台
-
-微信小游戏是微信小程序下的游戏产品平台，它不仅提供了强大的游戏能力，还和小程序一样，提供了大量的原生接口，比如支付，社交，文件系统，照片，NFC 等。相当于同时结合了 WEB 易于传播以及 Native 功能丰富的优势。
-
-小游戏的运行环境是小程序环境的扩展，基本思路也是封装必要的 WEB 接口提供给用户，尽可能追求和 WEB 同样的开发体验。小游戏在小程序环境的基础上提供了 WebGL 接口的封装，使得渲染能力和性能有了大幅度提升。不过由于这些接口都是微信团队通过自研的原生实现封装的，所以并不可以等同为浏览器环境。关于游戏方面，目前小游戏环境提供了 Canvas 以及 WebGL 的渲染接口，所以 Cocos Creator 引擎的两种渲染模式都是可以运行的，不过出于性能的考量，我们默认使用 WebGL 渲染游戏内容，强烈建议用户不要修改默认配置。
+## __WeChat__ Mini Game Platform
+__WeChat Mini Game__ is a game product platform used inside the __WeChat__ application. It not only provides powerful game capabilities, but also provides a large number of native interfaces, such as *payment*, *social*, *file system*, *photo*, *NFC*, and many more. It is equivalent to combining the advantages of the web for easy dissemination and rich native features.
 
-作为引擎方，为了尽可能简化开发者的工作量，我们为用户完成的主要工作包括：
+The running environment of the __WeChat Mini Game__ is an extension of the mini program environment. The basic idea is to provide the necessary web interface to the user, and pursue the same development experience as the web. The __WeChat Mini Game__ provides a wrapper around the WebGL interface based on the applet environment, which greatly improves the rendering capabilities and performance. However, since these interfaces are encapsulated by the __WeChat__ team they are not equivalent to the browser environment. Regarding the game aspect, the current __WeChat Mini Game__ environment provides the rendering interface of __Canvas__ and __WebGL__, so the two rendering modes of the __Cocos Creator__ engine can be run, but for performance reasons, we use __WebGL__ to render the game content by default, and users are strongly recommended. Do not modify the default configuration.
 
-- 引擎框架适配微信小游戏 API，纯游戏逻辑层面，用户不需要任何额外的修改
-- Cocos Creator 编辑器提供了快捷的打包流程，直接发布为微信小游戏，并自动唤起小游戏的开发者工具
-- 自动加载远程资源，缓存资源以及缓存资源版本控制
+As an engine, in order to simplify the developer's workload as much as possible, the main tasks we have done for our users include:
 
-除此之外，小游戏平台的游戏提交，审核和发布流程和小程序是没有区别的，都需要遵守微信团队的要求和标准流程，具体信息可以参考文末的链接。
+- The engine framework adapts to the WeChat Mini Game API, pure game logic level, users do not need any additional modifications
+- __Cocos Creator__ editor provides a fast packaging process, released directly as a WeChat game, and automatically evokes the developer tools of the game
+- Automatic loading of remote resources, cache resources and cache resource versioning
 
-## 使用 Cocos Creator 发布微信小游戏
+In addition, the game submission, review and release process and small program of the __WeChat Mini Game__ platform are indistinguishable. They all need to comply with the requirements and standard processes of the __WeChat__ team. For details, please refer to the link at the end of the article.
```

Comment:
In addition, the game submission, review and release process of the WeChat Mini Game platform is no different from the WeChat Mini Program.